components/LeaguePage/ClassScroll.tsx

```diff
@@ -85,18 +87,19 @@ export const ClassScroll: React.FC<ClassScrollProp> = ({
   if (classes.length == 0) return <Empty userType="Classes" />;
 
   return (
-    <>
+    <div>
       {filteredClasses.map((currClass: Class) => (
-        <ClassCard
-          key={currClass.eventInformationId}
-          name={currClass.name}
-          minLevel={currClass.minLevel}
-          maxLevel={currClass.maxLevel}
-          rrstring={currClass.rrstring}
-          startTime={currClass.startTime}
-          endTime={currClass.endTime}
-        />
+        <div onClick={() => router.push(`/class/${currClass.eventInformationId}`)}>
+          <ClassCard
+            name={currClass.name}
+            minLevel={currClass.minLevel}
+            maxLevel={currClass.maxLevel}
+            rrstring={currClass.rrstring}
+            startTime={currClass.startTime}
+            endTime={currClass.endTime}
+          />
+        </div>
       ))}
```

Comment:
this is really good, but we dont need this lol. These classes will be accessible through another page. You dont gotta worry about this

Reply:
Thanks for adding it and if its just for testing, lmk.

Reply:
yeah just for testing lol it was breaking lint errors anyways